AirportDictionary/AirportDictionary/Pages/Index.cshtml.cs

```diff
@@ -22,45 +24,44 @@ public IndexModel(ILogger<IndexModel> logger)
 
         public void OnGet()
         {
+            //requiredCity is the city for which we have to display the list of airports.
+            //This can be taken as a form input from the user.
+            string requiredCity = "Cincinnati";
+            List<Airport> airportsOfRequiredCity = new List<Airport>();
+            City newCity = new City();
 
-            using (var webClient = new System.Net.WebClient())
-            {
-
-                //IDictionary<long, QuickTypeCity.Welcome> allCities = new Dictionary<long, QuickTypeCity.Welcome>();
-
-                string cityJson = webClient.DownloadString("https://pkgstore.datahub.io/core/world-cities/world-cities_json/data/5b3dd46ad10990bca47b04b4739a02ba/world-cities_json.json");
-                QuickTypeCity.Welcome[] allCity = QuickTypeCity.Welcome.FromJson(cityJson);
-
-                ViewData["Welcome"] = allCity;
-
-                //foreach (QuickTypeCity.Welcome welcome in allCity)
-                //{
-                //    allCities.Add(welcome.Geonameid, welcome);
-
-                //}
-
-                // QuickTypeAirport.Airport[] allAirport = QuickTypeAirport.Airport.FromJson(airportJson);
-
-
-                string airportJson = webClient.DownloadString("https://pkgstore.datahub.io/core/airport-codes/airport-codes_json/data/552f9a534537e5a2f7a8b0584dad7e77/airport-codes_json.json");
-                var allAirport = QuickTypeAirport.Airport.FromJson(airportJson);
-
-                //JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("CitySchema.json"));
```

Comment:
Good to remove commented out code.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.